src/shared/codelens/localLambdaRunner.ts

```diff
@@ -191,7 +191,7 @@ export async function executeSamBuild({
 
     const samCliArgs: SamCliBuildInvocationArguments = {
         buildDir: samBuildOutputFolder,
-        baseDir: codeDir,
+        baseDir: path.dirname(inputTemplatePath),
```

Comment:
`executeSamBuild` should not have to infer any configuration. The values should be given to `executeSamBuild()` explicitly.